components/expenses/CreateExpenseForm.js

```diff
@@ -539,6 +539,17 @@ class CreateExpenseForm extends React.Component {
           <SignInOrJoinFree />
         </div>
       );
+    } else if (collective.isArchived) {
+      return (
+        <div className="CreateExpenseForm">
+          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph">
+            <FormattedMessage
+              id="expenses.create.archived"
+              defaultMessage="Cannot submit expenses for an archived collective. "
```

Comment:
You forgot the space at the end

```suggestion
              defaultMessage="Cannot submit expenses for an archived collective."
```